netci.groovy

```diff
@@ -181,31 +181,28 @@ branchList.each { branchName ->
                         shell("HOME=\$WORKSPACE/tempHome ./build.sh /p:ConfigurationGroup=${configurationGroup} /p:OSGroup=${osGroupMap[os]} /p:WithCategories=OuterLoop /p:TestWithLocalLibraries=true /p:ServiceUri=\$WcfServiceUri")
                     }
                 }
-                
-                // Set the affinity.  OS name matches the machine affinity.
-                if (os == 'Windows_NT') {
-                    // Set affinity for elevated machines on Windows
-                    Utilities.setMachineAffinity(newJob, os, 'latest-or-auto-elevated')
-                } 
-                else if (os == 'Ubuntu14.04') {
-                    Utilities.setMachineAffinity(newJob, os, "outer-latest-or-auto")
-                } 
-                else {
-                    Utilities.setMachineAffinity(newJob, os, 'latest-or-auto')
-                }
-            }
+            }                
 
-            // Disable the builds for non Windows_NT, since outerloops don't work yet
-            if (os != 'Windows_NT') 
-            {
-                newJob.disabled(true)
-            }            
+            // Set the affinity.  OS name matches the machine affinity.
+            if (os == 'Windows_NT') {
+                // Set affinity for elevated machines on Windows
+                Utilities.setMachineAffinity(newJob, os, 'latest-or-auto-elevated')
+            } 
+            else if (os == 'Ubuntu14.04') {
+                Utilities.setMachineAffinity(newJob, os, "outer-latest-or-auto")
+            } 
+            else {
+                Utilities.setMachineAffinity(newJob, os, 'latest-or-auto')
+            }
 
             // Set up standard options.
             Utilities.standardJobSetup(newJob, project, isPR, getFullBranchName(branchName))
             // Add the unit test results
             Utilities.addXUnitDotNETResults(newJob, 'bin/tests/**/testResults.xml')
             
+            // Due to limitations with how the CI server works, we have to disable concurrent builds for outerloops 
+            newJob.concurrentBuild(false)
```

Comment:
It seems like we need a better solution for this eventually. I can see lots of places where we would want to deploy multiple servers (developer scenarios for instance) concurrently.